modules/gapi/include/opencv2/gapi/imgproc.hpp

```diff
@@ -154,6 +154,13 @@ namespace imgproc {
             return in.withType(CV_8U, 1);
         }
     };
+
+    G_TYPED_KERNEL(Gnormalize, <GMat(GMat, double, double, int, int)>, "org.opencv.imgproc.normalize") {
```

Comment:
Actually cv::normalize is defined in core.hpp, so cv::gapi::normalize should resize in gapi/core.hpp